### PR TITLE
[JEP-5] - Jenkins Ambassador Program

### PR DESCRIPTION
**Pull request by bitwiseman**
@alyssat and I are working on getting the Jenkins Ambassador program going.  Starting this PR to gather early feedback.

### CODE REVIEW

**Comment by bitwiseman** [comment]
@jenkinsci/jep-editors This is a submission for approval as draft.  There's quite a bit more fleshing out of the details to be done, but it is draft ready.

File: jep/0000/README.adoc [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+The program will be largely self-governing with members making most decisions.
+CloudBees employees may be Ambassadors.
+There is no set limit to the number or percentage of CB employees that can be Ambassadors (however, see avoiding favoritism above).
+
+Jenkins Ambassadors operate as an independent group who use a consensus review process to select new members.
+All communication and discussion are conducted via the TBD (a public) mailing list.
+Jenkins Ambassadors receive via an email, the nominee's information based on the criteria above.
+Voting to select the nominee happens via the mailing list with either +1 or -1 from the peer group over a two-week period.
+If the nominee receives three +1 votes and zero negative votes by the end of two weeks then that person is confirmed as a Jenkins Ambassador.
+If there is a -1 vote then this triggers a discussion after which the negative vote may or may be changed. If the negative vote remains, the nominee cannot be confirmed.
```
[the remark this answers is not in the document]
Comment (oleg-nenashev) [reply jep/0000/README.adoc]: Does it mean that any Jenkins Ambassador can block all candidates from being elected during his/her term? Does not look good, people may go negative and toxic sometimes. I would rather prefer to see a vote override engine in this JEP.

File: jep/0000/README.adoc [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+If there are less than three +1 votes during the two-week voting period then further discussion needs to be encouraged.
+
+If there are less than three Ambassadors, the following Jenkins project contributors may cast votes:
```
Comment (oleg-nenashev) [inline jep/0000/README.adoc]: I would rather prefer a generic term like "Jenkins Board members and officers". Do not see why JEP would need to specify particular names if it is considered as a community program

File: jep/0000/README.adoc [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+offering support in forums, or organizing and contributing to JAMs and other events –
+they help make Jenkins better.
+
+
+=== Benefits to Ambassador
+
+* Recognition via jenkins.io
+* “Ambassador” blog highlighting their accomplishments
+* Free ticket to Jenkins World, JUC Israel
+* Staff Ask the Expert booth @ Jenkins World
```
Comment (oleg-nenashev) [inline jep/0000/README.adoc]: Sounds like rather a responsibility

File: jep/0000/README.adoc [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+* Free access to certifications
```
Comment (oleg-nenashev) [inline jep/0000/README.adoc]: You need to be more specific here. Which certifications? Jenkins ones or what?

**Comment by bitwiseman** [comment]
@oleg-nenashev 
Since you approved this as draft, could you go ahead and assign it a number and set it to draft per the usual approval process?

**Comment by oleg-nenashev** [comment]
@bitwiseman please do it if you want.
I would rather prefer to see responses in the dev list first

**Comment by bitwiseman** [comment]
@oleg-nenashev

I do not want. 😄 I submitted the PR it would be odd for me to do the "approval as draft".  

Responses on the Dev list are independent of "approval as draft".  You said, "The JEP draft looks good from the formatting part, so 👍 for [approval] it as a draft."

You are a [JEP Editor](https://github.com/jenkinsci/jep/blob/master/jep/1/README.adoc#editor) and you are the editor who approved this PR.  That means this JEP should be moved to draft status.

**Comment by oleg-nenashev** [comment]
Ack. I will handle it when I have time for it, likely late Tue or Wed.

**Comment by bitwiseman** [comment]
@oleg-nenashev  
Okay... It is has taken you longer to comment on this JEP and have this discussion than it would for you to do the approval.

**Pull request merged**